From 160bc5c672630ac1c1be585e02f636e90bbca87b Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Mon, 22 Jul 2019 22:43:21 -0700
Subject: [PATCH 0644/1852] v4l: xilinx: vpss-csc: Fix xcsc_set_format() for
 RBG 10 bpc

commit 0e7c314c67d8bfe2a288d6a5dbec02c2e19d5e9e from
https://github.com/Xilinx/linux-xlnx.git

This patch adds the missing RBG 10 bpc media bus format in the
xcsc_set_format().

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-vpss-csc.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/media/platform/xilinx/xilinx-vpss-csc.c b/drivers/media/platform/xilinx/xilinx-vpss-csc.c
index 2c79b6ec1b8b..c628ee07a1e0 100644
--- a/drivers/media/platform/xilinx/xilinx-vpss-csc.c
+++ b/drivers/media/platform/xilinx/xilinx-vpss-csc.c
@@ -787,6 +787,7 @@ static int xcsc_set_format(struct v4l2_subdev *subdev,
 	switch (__format->code) {
 	case MEDIA_BUS_FMT_VUY8_1X24:
 	case MEDIA_BUS_FMT_RBG888_1X24:
+	case MEDIA_BUS_FMT_RBG101010_1X30:
 	case MEDIA_BUS_FMT_UYVY8_1X16:
 	case MEDIA_BUS_FMT_VYYUYY8_1X24:
 	case MEDIA_BUS_FMT_VYYUYY10_4X20:
-- 
2.31.1

